src/main/java/com/toptrumps/db/PerformanceDAOImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.toptrumps.db;
+import java.util.Map;
+import java.sql.*; 
+import com.toptrumps.core.statistics.GameStateCollector;
+import com.toptrumps.core.player.Player;
+
+public class PerformanceDAOImpl implements GameDAO{
+
+    private int gameID;
+    private Map<Player,Integer> map;
+
+    public PerformanceDAOImpl(GameStateCollector gameState){
+
+        this.gameID = getGameID();
+        this.map = gameState.getRoundWinsMap();
+    }
+
+    public int getGameID(){
+
+        Connection conn = ConnectionFactory.getConnection();
```

Comment:
Just a note: in the ```GameDaoImpl``` you decided to name the ```Connection``` object as "con".

Reply:
Good catch! I have a bad habit of doing that haha.

Reply:
FFS nathan pay attention